docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "2.1"
+services:
+        postgres:
+                image: postgres:11-alpine
+                restart: always
+                volumes:
+                        - ${PWD}/data:/var/lib/postgresql/data
+                environment:
+                        POSTGRES_DB: ${FEATMAP_DB}
+                        POSTGRES_USER: ${FEATMAP_DB_USER}
+                        POSTGRES_PASSWORD: ${FEATMAP_DB_PASSWORD}
+                healthcheck:
+                        test: ["CMD-SHELL", "pg_isready -U postgres"]
```

Comment:
The user `postgres` does not exist, I think this needs to be changed to:

`test: ["CMD-SHELL", "pg_isready -U ${FEATMAP_DB_USER}"]`